docs/understanding-airbyte/secrets.md

```diff
@@ -55,8 +55,9 @@ Upon any operation, Airbyte reads the obfuscated JSON spec, extracts the secret
 
 ## Operational Details
 
-When configuration is updated, Airbyte increments a secret coordinate's version while preserving the prefix.
+1. When configuration is updated, Airbyte increments a secret coordinate's version while preserving the prefix.
 E.g. `airbyte_workspace_024eaa40-75ae-4014-941d-b9e938c3a363_secret_5675437d-ea6b-4f9a-9a0c-d76066004841_v1` -> `airbyte_workspace_024eaa40-75ae-4014-941d-b9e938c3a363_secret_5675437d-ea6b-4f9a-9a0c-d76066004841_v2`.
-
-This means secret coordinates contain two useful pieces of metadata to help operators. The workspace id allows operators to understand what workspace
+2. This means secret coordinates contain two useful pieces of metadata to help operators. The workspace id allows operators to understand what workspace
 a secret belongs to. The stable prefix allows operators to understand how often a secret has been modified and group secrets by connections.
+3. Not all secrets can be associated to a workspace e.g. Secrets created as part of a Discover or Check operation. These secrets have a workspace id of
```

Comment:
Maybe add the links for ./understanding-airbyte/airbyte-protocol/#check and ./understanding-airbyte/airbyte-protocol/#discover